main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/rollout/rox-go/v5/core/context"
+	"github.com/rollout/rox-go/v5/server"
+)
+
+type Flags struct {
+	ShowMessage server.RoxFlag
+	Message     server.RoxString
+	FontColor   server.RoxString
+	FontSize    server.RoxInt
+}
+
+var flags = &Flags{
+	ShowMessage: server.NewRoxFlag(false),
+	Message:     server.NewRoxString("This is the default message; try changing some flag values!", []string{}),
+	FontColor:   server.NewRoxString("Black", []string{"Red", "Green", "Blue", "Black"}),
+	FontSize:    server.NewRoxInt(12, []int{12, 16, 24}),
+}
+
+func main() {
+	router := gin.Default()
+	initFlags()
+
+	router.GET("/", func(c *gin.Context) {
+		c.JSON(http.StatusOK, gin.H{"message": "Hello, You Created a Web App!"})
+	})
+
+	router.GET("/demo", func(c *gin.Context) {
+		ctx := context.NewContext(map[string]interface{}{"email": "[email protected]"})
```

Comment:
I like that this shows an example of using a context for flag evaluation. But it seems to stop _just_ short of actually showing it in action.
e.g. if we added an `email` parameter to the request then you could use _that_ dynamic value in the context and have the flag show different values for different users.
How about having the simple GET endpoint like this evaluating a flag without a context (`IsEnabled(nil)`), and another POST endpoint that has a/some query/body parameter(s) that you then use in the flag context?